src/main.py

```diff
@@ -17,4 +25,17 @@ async def store_data(request: Request, token: dict = Depends(verify_jwt)):
         raise HTTPException(status_code=400, detail=str(e))
 
     await set_redis_from_tx(txn)
-    return {"message": "Data stored successfully!"}
+    return {"message": "success"}
+
+
+@app.on_event("startup")
+@repeat_at(cron="0 0 * * *")
+async def flush_db():
+    count = await redis_count()
+    print(f"{count} items found in redis")
```

Comment:
Instead of printing these, let's use Python's [`logging`](https://docs.python.org/3/library/logging.html), and set an overall logger with some settings.